tests/room/room.rs

```diff
@@ -1786,6 +1803,16 @@ mod patches_generation {
         });
 
         let commands: Vec<_> = command_rx.take(2).collect().await;
+
+        if let Ok(Ok(local_media_fail)) =
+            timeout(5000, Box::pin(media_fail_rx)).await
+        {
+            panic!(
+                "on_failed_local_media() triggered with error: {}",
+                local_media_fail
+            );
```

Comment:
Okay, so if I get this right, we have a flaky test and we can't reproduce why it fails, and this should help us find out whats happening when it eventually fails. In this case we should definitely add some TODO comment on whats going on here, and what to do when this fails. Otherwise everyone will just forget about this.